hpy/devel/include/common/hpydef.h

```diff
@@ -126,17 +126,34 @@ typedef struct {
         }                                           \
     };
 
-#define HPyDef_GET(SYM, NAME, IMPL, ...)                        \
-    static HPy IMPL(HPyContext ctx, HPy self, void *closure);   \
-    HPyFunc_TRAMPOLINE(SYM##_trampoline, IMPL, HPyFunc_GETTER); \
-    HPyDef SYM = {                                              \
-        .kind = HPyDef_Kind_GetSet,                             \
-        .getset = {                                             \
-            .name = NAME,                                       \
-            .getter_impl = IMPL,                                \
-            .getter_cpy_trampoline = SYM##_trampoline,          \
-            __VA_ARGS__                                         \
-        }                                                       \
+#define HPyDef_GET(SYM, NAME, IMPL, ...)                            \
+    static HPy IMPL(HPyContext ctx, HPy self, void *closure);       \
+    HPyFunc_TRAMPOLINE(SYM##_get_trampoline, IMPL, HPyFunc_GETTER); \
+    HPyDef SYM = {                                                  \
+        .kind = HPyDef_Kind_GetSet,                                 \
+        .getset = {                                                 \
+            .name = NAME,                                           \
+            .getter_impl = IMPL,                                    \
+            .getter_cpy_trampoline = SYM##_get_trampoline,          \
+            __VA_ARGS__                                             \
+        }                                                           \
+    };
+
+#define HPyDef_GETSET(SYM, NAME, GETIMPL, SETIMPL, ...)                     \
+    static HPy GETIMPL(HPyContext ctx, HPy self, void *closure);            \
+    static int SETIMPL(HPyContext ctx, HPy self, HPy value, void *closure); \
```

Comment:
same as above, `HPyFunc_DECLARE` ?

Reply:
Ah, right.  I wrote these macros before I realized I needed `HPyFunc_GETTER` and `HPyFunc_SETTER` anyway.  Will fix